app/models/concerns/vacancy_candidate_specification_validations.rb

```diff
@@ -13,7 +13,7 @@ module VacancyCandidateSpecificationValidations
   end
 
   def upload_feature_enabled?
```

Comment:
This method should not exist anymore

Reply:
I agree, making a PR for the candidate spec stuff to be removed following product approval